src/lib/checkbox/checkbox.ts

```diff
@@ -303,6 +303,10 @@ export class MatCheckbox extends _MatCheckboxMixinBase implements ControlValueAc
     this._changeDetectorRef.markForCheck();
   }
 
+  getAriaChecked(): string {
```

Comment:
Just also realized: method should be internal `_getAriaChecked`

The return type can really the string literal type `'true' | 'false' | 'mixed'`